### ISSUE TITLE
Please make an example

### ISSUE THREAD

**Issue by GeorgeFlorian** [revk/ESP32-PN532#1]
Hello.

I don't know if this is still maintained or not, but I would really love an example on how to use this library.

I am currently trying to integrate RFID1356MIFARE with ESP32-EVB but with no luck. 

Thank you.

**Comment by revk** [comment]
It is used in https://github.com/revk/SolarSystem which should give a good example.

**Comment by GeorgeFlorian** [comment]
> It is used in https://github.com/revk/SolarSystem which should give a good example.

I trust that it is, but I find it hard to follow. Can you point me to the right files ? Thank you.

**Comment by revk** [comment]
pn532_init is where you tell it the uart and pins. It is probably worth reading the include file to start with.

**Comment by revk** [comment]
I think that answers the question.
